content/docs/deployment/csmoperator/modules/authorization v2.0.md

```diff
@@ -1,8 +1,8 @@
 ---
-title: Authorization v2.0 Tech Preview
-linkTitle: "Authorization v2.0 Tech Preview"
+title: Authorization v2.0
+linkTitle: "Authorization v2.0"
 description: >
-  Installing Authorization v2.0 Tech Preview via Dell CSM Operator
+  Installing Authorization v2.0 via Dell CSM Operator
```

Comment:
Comment on the banner below - instead of publishing the karavictl deprecation which is already mentioned in 1.x pages, let's just indicate that karavictl is not supported with Auth 2.x and users should use dellctl instead. This need not be a banner on top, it can be a "Note" in the 'Installing dellctl' section below.